apps/authentication/migrations/0002_auto_20150304_2211.py

```diff
@@ -7,10 +7,7 @@
 
```

Comment:
I don't think it's that smart to change migrations.

Removing the dependency on this migration doesn't remove the tables/columns from the database. A separate migration is required to do this. After that migration is done, a new change can be done where the requirement is removed, and migrations can be squashed to remove the dependency.

Reply:
I just saw that you've done it correctly with the sso app migration, good. But I still think this migration shouldn't be changed (and therefore all the sso migrations have to be kept), until it is squashed so that sso is fully removed. It's a bit of a long run, but I don't think it matters that much to keep the migrations in for a while. Maybe @duvholt could chime in on this?

Reply:
@sklirg I see! So it would be better to not involve this and perhaps add something like it in it's own PR for example?